awsbestpractices/bad-params/BadParams.java

```diff
@@ -0,0 +1,21 @@
+package example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.services.elasticmapreduce.model.ActionOnFailure;
+
+public class BadParamsTestCases {
+
+    public void badParams() {
+        // Should flag
+        new StepConfig().withName(url).withActionOnFailure("TERMINATE_JOB_FLOW");
+
+        // Should flag
+        new StepConfig().withName(url).withActionOnFailure(ActionOnFailure.TERMINATE_JOB_FLOW);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses an outdated value. Consider using the recommended value instead. Recommended value: TERMINATE_CLUSTER